src/olympia/activity/views.py

```diff
@@ -48,6 +51,8 @@ class VersionReviewNotesViewSet(
 
     def get_queryset(self):
         alog = ActivityLog.objects.for_versions(self.get_version_object())
+        if not acl.is_user_any_kind_of_reviewer(self.request.user):
```

Comment:
pass `allow_viewers=False` too

Reply:
Confirmed via Matrix that you meant `allow_viewers=True`. 😄